index.js

```diff
@@ -0,0 +1,7 @@
+require('dotenv').config();
```

Comment:
I'd do this in server.js instead, so you can be sure it will run first than everything else.